tracer/test/snapshots/Security.AspNetCore5AsmRemoteRules._.verified.txt

```diff
@@ -89,6 +90,8 @@
       process_id: 0,
       _dd.agent_psr: 1.0,
       _dd.appsec.enabled: 1.0,
+      _dd.appsec.event_rules.error_count: 0.0,
+      _dd.appsec.event_rules.loaded: 1.0,
```

Comment:
Are these additions expected? 🤔

Reply:
yes! they should have been there all along.
It's because when the waf is reinitialized with a new ruleset, these will potentially change

Reply:
and this test gives new rules to the waf, which means waf is disposed and reinitialized (how the waf works atm)